### PR TITLE
podsync 2.5.0 (new formula)

### PR DESCRIPTION
**Pull request by nathanmsmith**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Add a new formula for [podsync](https://github.com/mxpv/podsync), a self-hostable service that that lets you listen to any YouTube / Vimeo channels, playlists or user videos in podcast format.

### CODE REVIEW

File: Formula/podsync.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        [storage.local]
+        data_dir = "data/podsync/"
+
+      [feeds]
+        [feeds.ID1]
+        url = "https://www.youtube.com/channel/UCxC5Ls6DwqV0e-CYcAKkExQ"
+    EOS
+
+    pid = fork do
+      exec "#{bin}/podsync"
```
Comment (SMillerDev) [inline Formula/podsync.rb:38]:
```suggestion
      exec bin/"podsync"
```

File: Formula/podsync.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+class Podsync < Formula
+  desc "Turn YouTube or Vimeo channels, users, or playlists into podcast feeds"
+  homepage "https://github.com/mxpv/podsync"
+  url "https://github.com/mxpv/podsync/archive/refs/tags/v2.5.0.tar.gz"
+  sha256 "45c3afea38ef45f665f456bc542ac57fecdafed884181156df6b44624b66d6b8"
+  license "MIT"
+  head "https://github.com/mxpv/podsync.git", branch: "main"
+
+  depends_on "ffmpeg"
+  depends_on "go"
```
Comment (dawidd6) [inline Formula/podsync.rb:10]: Is Go needed at runtime?

Comment (nathanmsmith) [reply Formula/podsync.rb]: Good question! The readme was ambiguous, but to me [implied yes](https://github.com/mxpv/podsync#dependencies). Logically, I'd think you wouldn't need it once you have the binary.

I just tested `podsync` after uninstalling `go` and everything worked perfectly. Moving to a build dep.

**Review by dawidd6 — APPROVED** [review]

**Comment by p-linnane** [comment]
Thank you for your contribution to `homebrew-core` @nathanmsmith!

**Pull request closed without merging**